EventEmitter.d.ts

```diff
@@ -19,30 +19,30 @@ declare namespace EventEmitter {
         [event: string]: Listener[];
     }
 
-    type Listeners = Listener[]|ListenerMap;
+    type Listeners = Listener[] | ListenerMap;
 
-    export class EventEmitter {
+    export class EventEmitter<E = EventKey> {
```

Comment:
Yes it is ture. If no generic type if given, `EventKey` is used. But is there is any given, `E` will turn to need the given value. Typescript genric works like any other generic languages.

I' ll test providing no value to the generic type resulting is need a type of `KeyEvent`and providing eg. an enum type will result in requireing a typeof of enum given to the generic pattern.

You have to know that this default type `KeyEvent` will only allow a backwards compatiblity at typescript >= 2.3. but typescript ^2.4 is at current stable.

I hope this meight help